<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1430</width>
    <height>905</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/new/prefix1/33.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1411</width>
      <height>801</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1430</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
    <addaction name="action_20"/>
    <addaction name="action_22"/>
    <addaction name="action_21"/>
    <addaction name="action_23"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_17"/>
    <addaction name="action_12"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
   <addaction name="action_7"/>
   <addaction name="action_8"/>
   <addaction name="action_9"/>
   <addaction name="action_15"/>
   <addaction name="action_20"/>
   <addaction name="action_22"/>
   <addaction name="action_21"/>
   <addaction name="action_23"/>
   <addaction name="action_14"/>
   <addaction name="action_17"/>
   <addaction name="action_12"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片18.png</normaloff>:/new/prefix1/图片18.png</iconset>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片3.png</normaloff>:/new/prefix1/图片3.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片4.png</normaloff>:/new/prefix1/图片4.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片22.png</normaloff>:/new/prefix1/图片22.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片1.png</normaloff>:/new/prefix1/图片1.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>撤销Ctrl+Z</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片17.png</normaloff>:/new/prefix1/图片17.png</iconset>
   </property>
   <property name="text">
    <string>剪切Ctrl+X</string>
   </property>
   <property name="toolTip">
    <string>剪切Ctrl+X</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片19.png</normaloff>:/new/prefix1/图片19.png</iconset>
   </property>
   <property name="text">
    <string>复制Ctrl+C</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片20.png</normaloff>:/new/prefix1/图片20.png</iconset>
   </property>
   <property name="text">
    <string>粘贴Ctrl+V</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片24.png</normaloff>:/new/prefix1/图片24.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片23.png</normaloff>:/new/prefix1/图片23.png</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片16.png</normaloff>:/new/prefix1/图片16.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>全选Ctrl+A</string>
   </property>
  </action>
  <action name="action_14">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片14.png</normaloff>:/new/prefix1/图片14.png</iconset>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片5.png</normaloff>:/new/prefix1/图片5.png</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="action_17">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片15.png</normaloff>:/new/prefix1/图片15.png</iconset>
   </property>
   <property name="text">
    <string>关于记事本</string>
   </property>
  </action>
  <action name="action_18">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>字体倾斜</string>
   </property>
  </action>
  <action name="action_19">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="action_20">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片8.png</normaloff>:/new/prefix1/图片8.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="action_21">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片9.png</normaloff>:/new/prefix1/图片9.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="action_22">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片7.png</normaloff>:/new/prefix1/图片7.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="action_23">
   <property name="icon">
    <iconset resource="../../QtPicture/pic.qrc">
     <normaloff>:/new/prefix1/图片10.png</normaloff>:/new/prefix1/图片10.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../QtPicture/pic.qrc"/>
 </resources>
 <connections/>
</ui>
